src/Altinn.AccessManagement.Persistence/Migration/v0.08-accessmanagement/01-setup-functions.sql

```diff
@@ -0,0 +1,52 @@
+-- FUNCTION: CREATE new delegation.select_active_resourceregistrydelegationchanges_admin
+CREATE OR REPLACE FUNCTION delegation.select_active_resourceregistrydelegationchanges_coveredbypartys(
```

Comment:
Ahh, you actually need both OfferedBy and CoveredBy partyIds to be optional?
Then I agree this should be a new function and not change/variant of the existing select_active_resourceregistrydelegationchanges_coveredbypartys (which requires CoveredByPartyId)

So you should name it something specific e.g. select_active_resourceregistrydelegationchanges_admin as per the -- FUNCTION: CREATE comment 

sorry for the confusion from my previous comments on this.